src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilarIssues/similarStackTrace/toolbar.jsx

```diff
@@ -1,76 +0,0 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import createReactClass from 'create-react-class';
-import Reflux from 'reflux';
-
-import {t} from 'app/locale';
-import GroupingStore from 'app/stores/groupingStore';
-import SpreadLayout from 'app/components/spreadLayout';
-import FlowLayout from 'app/components/flowLayout';
-import LinkWithConfirmation from 'app/components/links/linkWithConfirmation';
-import Toolbar from 'app/components/toolbar';
-import ToolbarHeader from 'app/components/toolbarHeader';
-
-const SimilarToolbar = createReactClass({
-  displayName: 'SimilarToolbar',
-
-  propTypes: {
-    onMerge: PropTypes.func.isRequired,
-  },
-
-  mixins: [Reflux.listenTo(GroupingStore, 'onGroupingUpdate')],
-
-  getInitialState() {
-    return {
-      mergeCount: 0,
-    };
-  },
-
-  onGroupingUpdate({mergeList}) {
-    if (mergeList && mergeList.size !== this.state.mergedCount) {
-      this.setState({
-        mergeCount: mergeList.size,
-      });
-    }
-  },
-
-  render() {
-    const {onMerge} = this.props;
-    return (
-      <Toolbar className="similar-toolbar">
```

Comment:
Are we able to delete the CSS classes that this was using?

Reply:
this selector did not exist.